project_deadline/tests/test_project_deadline.py

```diff
@@ -14,7 +14,5 @@ def test_fields_are_appended(self):
     def test_other_views(self):
         view = self.env.ref("project.edit_project")
         res = self.env["project.project"].fields_view_get(view.id, "tree")
-
-        # It should not affect other view types
-        self.assertNotIn("date_start", res["arch"])
-        self.assertNotIn("date", res["arch"])
+        self.assertIn("date_start", res["arch"])
+        self.assertIn("date", res["arch"])
```

Comment:
**question**: why were these two lines changed to be the opposite of what they were before?

Reply:
As part of Odoo 16.0 addons project view, date_start and date fields are included in the xml whose external id is "edit_project" which is displayed as "Planned Date". 

https://github.com/odoo/odoo/blob/16.0/addons/project/views/project_views.xml#L506C28-L511C119


![Screenshot from 2024-10-22 16-45-36](https://github.com/user-attachments/assets/62d751eb-e6f4-4dfe-b128-6dc4ca66c1ba)

 
 As this is available as part of Odoo addons project module, I am removing project_project related code changes which appends date_start and date field to the form view from project_deadline module.
 Also, the testcase related to it is also no longer valid for 16.0 version